omnigibson/robots/tiago.py

```diff
@@ -499,7 +499,9 @@ def camera_control_idx(self):
 
     @property
     def arm_control_idx(self):
-        return {"left": np.array([7, 10, 13, 15, 17, 19, 21]), "right": np.array([8, 11, 14, 16, 18, 20, 22])}
+        return {"left": np.array([7, 10, 13, 15, 17, 19, 21]),
+                "right": np.array([8, 11, 14, 16, 18, 20, 22]),
+                "combined": np.array([7, 8, 10, 11, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22])}
```

Comment:
@mj-hwang's FK work. Could you explain why?

Reply:
@mj-hwang

Reply:
This combined mode is used when we do forward kinematics for mesh overlap during arm motion planning. Previously, we only figured out left arm and trunk link poses (using "tiago_dual_omnidirectional_stanford_left_arm_descriptor.yaml"), but we weren't able to get the correct link poses for right arm links, since FKSolver would only return default link poses for the uncontrolled arm. For the same reason, I updated the "robot_arm_descriptor_yamls" function and the configuration file in assets ("tiago_dual_omnidirectional_stanford_descriptor.yaml").

Reply:
I see, interesting. This bimanual setup does raise quite a lot of headaches. SGTM.